po/POTFILES

```diff
@@ -15,31 +15,31 @@ src/app/components/user_menu/user_menu.rs
 src/app/state/login_state.rs
 src/main.rs
 
-# find src -name "*.ui*" -print
+# find src -name "*.ui*" -o -name "*.blp*"
 src/window.ui
```

Comment:
.blp now right? :tada: 

(also would you mind leaving the po files in their previous state? I regenerate them from poeditor every one in a while, I'll probably do it soon anyway to make sure nothing broke as it is a bit hard to review them here)

Reply:
Yes, but I have kept *.ui* in the find command in case we need to use .ui files again.

Reply:
I'll rebase my PR and remove the edits I made to .pot and .po files once I get home.

Reply:
sorry I meant the `src/window.ui` line (19)

(the find command was there just to help me remember how to quickly list all ui (now blp) files, we might not need to have a command that helps with porting here, but it's fine)